src/index.ts

```diff
@@ -49,4 +54,4 @@ const waitForExpect = function waitForExpect(
   });
 };
 
-export default waitForExpect;
+export default Object.assign(waitForExpect, { defaults });
```

Comment:
I attached `defaults` to the exported function rather than as a separate export so that it gets passed along for free when it's re-exported in `react-testing-library` or similar